src/sparseml/pytorch/image_classification/train.py

```diff
@@ -16,72 +16,79 @@
+                                           [--model-kwargs MODEL_KWARGS]
+                                           [--dataset-kwargs DATASET_KWARGS]
+                                           [--model-tag MODEL_TAG]
+                                           [--save-dir SAVE_DIR]
+                                           [--device DEVICE]
+                                           [--loader-num-workers LOADER_NUM_WORKERS]
+                                           [--no-loader-pin-memory]
+                                           [--loader-pin-memory [LOADER_PIN_MEMORY]]
+                                           [--image-size IMAGE_SIZE]
 
 optional arguments:
   -h, --help            show this help message and exit
   --train-batch-size TRAIN_BATCH_SIZE
                         The batch size to use while training
   --test-batch-size TEST_BATCH_SIZE
                         The batch size to use while testing
-  --arch-key ARCH_KEY   The type of model to use, ex: resnet50, vgg16,
-                        mobilenet put as help to see the full list (will raise
-                        an exception with the list)
   --dataset DATASET     The dataset to use for training, ex: imagenet,
                         imagenette, cifar10, etc. Set to imagefolder for a
                         generic dataset setup with an image folder structure
                         setup like imagenet or loadable by a dataset in
                         sparseml.pytorch.datasets
   --dataset-path DATASET_PATH
                         The root path to where the dataset is stored
+  --arch-key ARCH_KEY   The type of model to use, ex: resnet50, vgg16,
+                        mobilenet put as help to see the full list (will raise
+                        an exceptionwith the list)
   --checkpoint-path CHECKPOINT_PATH
                         A path to a previous checkpoint to load the state from
                         and resume the state for. If provided, pretrained will
-                        be ignored. If using a SparseZoo recipe, can also
+                        be ignored . If using a SparseZoo recipe, can also
                         provide 'zoo' to load the base weights associated with
                         that recipe
   --init-lr INIT_LR     The initial learning rate to use while training, the
                         actual initial value used should be set by the
                         sparseml recipe
   --optim-args OPTIM_ARGS
                         Additional args to be passed to the optimizer passed
-                        in as a json object
+                        in as a json object. Defaults set for SGD
   --recipe-path RECIPE_PATH
                         The path to the yaml file containing the modifiers and
                         schedule to apply them with. Can also provide a
                         SparseZoo stub prefixed with 'zoo:' with an optional
                         '?recipe_type=' argument
```

Comment:
```suggestion
                        'recipe_type=' argument
```
what does this "?" stand for?